pkg/scheduler/objects/allocation_test.go

```diff
@@ -232,12 +232,25 @@ func TestNewAllocFromSI(t *testing.T) {
 	var nilAlloc *Allocation
 	alloc := NewAllocationFromSI(allocSI)
 	assert.Equal(t, alloc, nilAlloc, "placeholder allocation created without a TaskGroupName")
-	allocSI.TaskGroupName = "testgroup"
```

Comment:
Maybe we can simplify replace the string by another one. For example: `allocSI.TaskGroupName = "TestNewAllocFromSI"`